src/com/github/argajuvi/Main.java

```diff
@@ -104,11 +102,15 @@ public void loginUser(List<User> users) {
                         // VALIDASI APAKAH LOGIN UNTUK ADMIN ATAU USER
                         if (username.equals("admin")) {
                             // SHOW MENU ADMIN KALAU YANG LOGIN ADMIN
-                            this.showAdminMenu();
+//                            this.showAdminMenu();
+                            Menu adminMenu = new AdminMenu();

```

Comment:
Kenapa gk buat variable `Menu menu;` aja di luar `if-else` ini? It's cleaner that way m8